client/src/components/Payment/StripeForm.tsx

```diff
@@ -43,9 +56,31 @@ const StripeForm = ({ onClose }: Props): JSX.Element => {
     const { error, paymentMethod } = await stripe.createPaymentMethod({
       type: 'card',
       card: cardElement,
+      billing_details: billingDetails,
```

Comment:
Change to `billingDetails`

Reply:
Hi, the billing_details is the property of stripe.createPaymentMethod method.

Reply:
OK. That's fine!